README.md

```diff
@@ -12,20 +12,36 @@ Cell-by-cell testing for production Jupyter notebooks in JupyterLab
 # Overview
 `Celltests` is designed for writing tests for linearly executed notebooks. Its primary use is for unit testing reports. 
 
-### "Linearly executed notebooks?"
+## Installation
+Python package installation:
+
+`pip install nbcelltests`
+
+or from source:
+
+`python setup.py install`
+
+To install the frontend extension for JupyterLab
+
+```bash
+jupyter labextension install jupyterlab_celltests
+jupyter serverextension enable --py nbcelltests
```

Comment:
We should probably improve the naming at some point :)

Reply:
I actually think this works. we had a conversation previously about separating the backend and frontend components, I think the backend is called nbcelltests and the one instance of the frontend that works with jupyterlab is called jupyterlab_celltests. If we integrate with VS Code, it will be called vscode_celltests, if we support classic notebook, jupyter_celltests, etc.